internal/venice-common/src/main/java/com/linkedin/venice/acl/handler/AbstractStoreAclHandler.java

```diff
@@ -160,7 +153,13 @@ protected AccessResult checkAccess(String uri, X509Certificate clientCert, Strin
         // Requested resource exists but does not have ACL.
         // Action:
         // return 401 Unauthorized
-        LOGGER.warn("Requested store does not have ACL: {}", errLine);
+        if (LOGGER.isWarnEnabled()) {
+          LOGGER.warn(
+              "Requested store does not have ACL: {} requested {} {}",
+              identityParser.parseIdentityFromCert(clientCert),
+              method,
+              uri);
+        }
         LOGGER.debug(
```

Comment:
Do we want to check for `isDebugEnabled` for these lines too?

Reply:
Not strictly necessary, but just to keep the style consistent, I added it systematically throughout this function, in my latest commit.